content/_code-samples/build-a-wallet/desktop-js/7-send-xrp/view/template.html

```diff
@@ -0,0 +1,156 @@
+
+        <div class="divider"></div>
+
+        <div class="main-content tab-content d-flex flex-column flex-shrink-0 p-3">
+
+            <div class="header border-bottom">
+                <h3>
+                    Build a XRPL Wallet
+                    <small class="text-muted">- Part 7/8</small>
+                </h3>
+                <button type="button" class="btn btn-primary" id="send-xrp-modal-button">
+                    Send XRP
+                </button>
+            </div>
+
+            <div class="tab-pane fade show active" id="dashboard" role="tabpanel" aria-labelledby="dashboard-tab">
+                <h3>Account:</h3>
+                <ul class="list-group">
+                    <li class="list-group-item">Classic Address: <strong id="account-address-classic"></strong></li>
+                    <li class="list-group-item">X-Address: <strong id="account-address-x"></strong></li>
+                    <li class="list-group-item">XRP Balance: <strong id="account-balance"></strong></li>
+                </ul>
+                <div class="spacer"></div>
+                <h3>
+                    Ledger
+                    <small class="text-muted">(Latest validated ledger)</small>
+                </h3>
+                <ul class="list-group">
+                    <li class="list-group-item">Ledger Index: <strong id="ledger-index"></strong></li>
+                    <li class="list-group-item">Ledger Hash: <strong id="ledger-hash"></strong></li>
+                    <li class="list-group-item">Close Time: <strong id="ledger-close-time"></strong></li>
+                </ul>
+            </div>
+
+            <div class="tab-pane fade" id="transactions" role="tabpanel" aria-labelledby="transactions-tab">
+                <h3>Transactions:</h3>
+                <table id="tx-table" class="table">
+                    <thead>
+                        <tr>
+                            <th>Confirmed</th>
+                            <th>Type</th>
+                            <th>From</th>
+                            <th>To</th>
+                            <th>Value Delivered</th>
+                            <th>Hash</th>
+                        </tr>
+                    </thead>
+                    <tbody></tbody>
+                </table>
+            </div>
+
+        </div>
+
+        <div class="modal fade" id="send-xrp-modal" tabindex="-1">
+            <div class="modal-dialog">
+                <div class="modal-content">
+                    <div class="modal-header">
+                        <h1 class="modal-title fs-5" id="send-xrp-modal-label">Send XRP</h1>
+                        <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+                    </div>
+                    <div class="modal-body">
+                        <div class="input-group mb-3">
+                            <input type="text" class="form-control" value="rP4zcp52pa7ZjhjtU9LrnFcitBUadNW8Xz"
```

Comment:
For consistency with the version you build live, this should use the placeholder with r9j...

```suggestion
                            <input type="text" class="form-control" placeholder="r9jEyy3nrB8D7uRc5w2k3tizKQ1q8cpeHU"
```